lib/galaxy/web/base/interactive_environments.py

```diff
@@ -184,18 +189,19 @@ def docker_cmd(self, env_override={}, volumes=[]):
         conf.update(env_override)
         env_str = ' '.join(['-e "%s=%s"' % (key.upper(), item) for key, item in conf.items()])
         volume_str = ' '.join(['-v "%s"' % volume for volume in volumes])
+        import_volume_str = '-v "{temp_dir}:/import/"'.format(temp_dir=temp_dir) if self.attr.import_volume else ''
```

Comment:
why not add this to the `volumes` list? Then we could have the formatting done for us rather than this complex line.